tmux-cssh

```diff
@@ -67,6 +67,21 @@ if [ -n "${tmux_attach_current_session}" ]; then
     tmux_window_options="-n ${tmux_window}"
 fi
 
+# If host doesn't look like a DNS name, it may be a CSSH cluster
+if [[ $hosts != *.* && $hosts != *" "* ]]; then
```

Comment:
`[[` is a bashism, can you update this to stay POSIX sh compliant please?

Reply:
Good point - fixed :)